### ISSUE TITLE
[Balance] Alien players have extremely weak defense

### ISSUE THREAD

**Issue by Stooberton** [Unvanquished/Unvanquished#1244]
1) Aliens cannot evolve when humans are nearby, leaving them stuck as dretches and unable to effectively defend the base/outpost. Possible solution here would be to allow aliens to evolve when nearby the overmind all the time, even with humans nearby.
2) Aliens spawn automatically at the closest egg, even if there are humans very close by, this almost always results in a respawned player dying instantly. This applies to humans too, neither should work with enemies nearby if there are other spawns available OR the nearest egg is near the overmind.
3) Aliens heal extremely slowly, much slower than humans, even inside their own base. It is _extremely easy_ to run inside an alien base as a human, kill the alien, and run out.  Any alien that defends their base can only do so once before they have to heal for upwards of 30 seconds. This would be fixed by something like the Overmind always providing the healing of a booster.

**Comment by illwieckz** [comment]
1. Upcoming release makes possible for aliens to evolve when nearby, well, they can evolve if not in direct sight and a close range, making far easier to evolve, and prevent the annoying situation of being prevented to evolve while being right behind the wall of a human base but no one sees you, or you're in alien base and humans are close but you're somewhat safe. Note: Upcoming release will allow do devolve next to overmind (cost for such devolve would be related to your current health, the more healthier you are the less it costs you to devolve).
2. On the other hand leaving creeps unprotected can be bad, to be honest there is no easy change to do on that part. Let's look at this example: an alien egg high in plat23 center part (platform), aliens can maintain humans between the center part and their indoor part while safely spawning if the other teammates does the job well, with your suggestion the aliens would just be spawned far in base, killing the pressure they apply on preventing humans to leave their base and leaving that strategic egg unprotected… Somebody talks about being able to choose the spawn point… maybe that can help.
3. Yes, that's a really big problem. Lone aliens heal very slowly, maybe too much slowly. Note that any alien creep can heal (less than booster but significant enough), so spawning some hidden drill for example will not only brings build point but give aliens a way to heal. This mechanism encourage territory control and forward base building. I don't feel that creep heal and overmind heal are too slow, I'm more considered by the self-healing lowness. Note that balancing the teams not necessarily means giving them the same abilities (that would be a bit killing the specificity of the game). One another issue that is reported by players is that human players can run as fast as aliens, even backward. Maybe making aliens able to have more chance to save their lives and having more chance to finish a wounded humans would balance the fast human healing rate ? This or something, it may be more interesting to balance some ability by others ability, not just by ironing out everything flat.

TL;DR: we have to find ways to tell players about those mechanism, like building any alien creep near human base can can help maintaining pressure on human base by providing better healing, while waiting for more efficient booster.

**Comment by Gireen** [comment]
1.  is solved in upcoming release as illwieckz
2. #1232 its probably a matter of personal preference if you want to spawn as close as possible to the enemy or in a safer distance. i for one prefer the close version except if the enemys have luci battlesuits, but then every hope is lost anyway. 🙈
3. yes. alien regeneration and defense structures need maybe some changes. it happens to often that aliens have to watch humans attack there base without much they can do.
